---a/net/minecraft/util/EnumTypeAdapterFactory.java
+++b/net/minecraft/util/EnumTypeAdapterFactory.java
@@ -1,5 +1,9 @@
 package net.minecraft.util;
 
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Locale;
+
 import com.google.common.collect.Maps;
 import com.google.gson.Gson;
 import com.google.gson.TypeAdapter;
@@ -8,57 +8,42 @@
 import com.google.gson.stream.JsonReader;
 import com.google.gson.stream.JsonToken;
 import com.google.gson.stream.JsonWriter;
-import java.io.IOException;
-import java.util.HashMap;
-import java.util.Locale;
 
-public class EnumTypeAdapterFactory implements TypeAdapterFactory
-{
+public class EnumTypeAdapterFactory implements TypeAdapterFactory {
+
     private static final String __OBFID = "CL_00001494";
 
-    public TypeAdapter create(Gson p_create_1_, TypeToken p_create_2_)
-    {
+    public TypeAdapter create(Gson p_create_1_, TypeToken p_create_2_) {
         Class oclass = p_create_2_.getRawType();
 
-        if (!oclass.isEnum())
-        {
+        if (!oclass.isEnum()) {
             return null;
-        }
-        else
-        {
+        } else {
             final HashMap hashmap = Maps.newHashMap();
             Object[] aobject = oclass.getEnumConstants();
             int i = aobject.length;
 
-            for (int j = 0; j < i; ++j)
-            {
+            for (int j = 0; j < i; ++j) {
                 Object object = aobject[j];
                 hashmap.put(this.func_151232_a(object), object);
             }
 
-            return new TypeAdapter()
-            {
+            return new TypeAdapter() {
+
                 private static final String __OBFID = "CL_00001495";
-                public void write(JsonWriter p_write_1_, Object p_write_2_) throws IOException
-                {
-                    if (p_write_2_ == null)
-                    {
+
+                public void write(JsonWriter p_write_1_, Object p_write_2_) throws IOException {
+                    if (p_write_2_ == null) {
                         p_write_1_.nullValue();
-                    }
-                    else
-                    {
+                    } else {
                         p_write_1_.value(EnumTypeAdapterFactory.this.func_151232_a(p_write_2_));
                     }
                 }
-                public Object read(JsonReader p_read_1_) throws IOException
-                {
-                    if (p_read_1_.peek() == JsonToken.NULL)
-                    {
+
+                public Object read(JsonReader p_read_1_) throws IOException {
+                    if (p_read_1_.peek() == JsonToken.NULL) {
                         p_read_1_.nextNull();
                         return null;
-                    }
-                    else
-                    {
+                    } else {
                         return hashmap.get(p_read_1_.nextString());
                     }
                 }
@@ -66,10 +66,8 @@
         }
     }
 
-    private String func_151232_a(Object p_151232_1_)
-    {
-        return p_151232_1_ instanceof Enum ? ((Enum)p_151232_1_).name().toLowerCase(Locale.US) : p_151232_1_.toString().toLowerCase(Locale.US);
+    private String func_151232_a(Object p_151232_1_) {
+        return p_151232_1_ instanceof Enum ? ((Enum) p_151232_1_).name().toLowerCase(Locale.US) : p_151232_1_.toString().toLowerCase(Locale.US);
     }
-}
+}
